### ISSUE TITLE
UX mockup available

### ISSUE THREAD

**Issue by tuscland** [probabl-ai/skore#768]

**Comment by anasstarfaoui** [comment]
https://www.figma.com/design/Qzjyxv0lPDU2kKtsH9dcp3/Skore---Workstation---High-fidelity-mock-ups---Q3?node-id=4518-57&t=wXnEwF51k7uB7IYW-4

Here is the link to current explorations, awaiting for initial feedbacks to continue.

**Comment by tuscland** [comment]
cc @MarieS-WiMLDS, I think we can start with the lower rightmost design. WDYT?

**Comment by anasstarfaoui** [comment]
The design is good for review; 

https://www.figma.com/design/Qzjyxv0lPDU2kKtsH9dcp3/Skore---Workstation---High-fidelity-mock-ups---Q3?node-id=4609-7164&t=wXnEwF51k7uB7IYW-4

@tuscland @MarieS-WiMLDS

**Comment by anasstarfaoui** [comment]
Yes tell me, do you think it could be triggering? I added it to represent "time", but not extremely attached to it

**Comment by MarieS-WiMLDS** [comment]
It was the question about the "comma" related to the vertical bar, but you answered it already in figma :) !

**Comment by tuscland** [comment]
Once outstanding comments are solved, we can copy/paste the resulting design in the parent story, and close this issue.
